app/react/App/scss/layout/_tableview.scss

```diff
@@ -110,7 +108,3 @@
     }
   }
 }
-
-.library-viewer.no-scroll-layout {
```

Comment:
I've attached a screenshot that has a visual bug that might be caused by removing this line.

Reply:
![image](https://user-images.githubusercontent.com/71732018/189416614-2799238a-b9d2-4eef-98b2-7fd02eb241b7.png)

Reply:
Great catch, thank you @Zasa-san